### PR TITLE
Feature/postgres operator

### PR DESCRIPTION
**Pull request by sonamsamdupkhangsar**

### CODE REVIEW

File: charts/service/values.yaml [hunk 1]
```diff
@@ -93,6 +93,14 @@ istioAllowedMethods: []
 #   replicationFactor: 2 
 #   envKey: SOME_TOPIC_NAME_DEAD_LETTER
```
Comment (fuzzybaird) [inline charts/service/values.yaml:95]: Is it widely known what a team-id is? Maybe we should reference somewhere what that value is?

File: charts/service/values.yaml [hunk 2]
```diff
@@ -93,6 +93,14 @@ istioAllowedMethods: []
...
 
+# postgres:
+#  teamId: sa
+#  dbName: demodb
+#  owner: saadmin
+#  version: 11  # database version
+#  size: 1Gi   # database size 
+#  numberOfInstances: 2 # number of postgres instances 1 - master 2..replica
```
Comment (happiness801) [inline charts/service/values.yaml:102]: Great job adding some comments to help users make a choice.

File: charts/service/values.yaml (hunk 1 above)
Comment (paraplexed) [reply charts/service/values.yaml:95]: Let's just add that to the documentation here maybe?
https://chghealthcare.atlassian.net/wiki/spaces/IEA/pages/166133960/Postgres

File: charts/service/values.yaml (hunk 2 above)
Comment (sonamsamdupkhangsar) [reply charts/service/values.yaml:102]: Thanks.

File: charts/service/values.yaml (hunk 1 above)
Comment (sonamsamdupkhangsar) [reply charts/service/values.yaml:95]: I will add the information in the confluence page and what the document refers to teamId.

**Pull request merged**